Makefile

```diff
@@ -85,3 +86,11 @@ format:
 	@echo "format code with google style"
 	yapf -i -r ./qingstor ./tests ./scenarios
 	@echo "ok"
+
+offline-package: build
+	@echo "generate a gzip file for offline installation"
+	mkdir dist/depedence
+	pip download qsctl -d dist/depedence
+	tar -czvf dist/qsctl-offline-1.6.2.tar.gz dist/depedence
```

Comment:
And this could add into `release-xxx` command, for windows user, they need to use zip.